experiments/jz/entity/exp_2/toy_example/03_create_dataset.slurm

```diff
@@ -47,9 +47,9 @@ REPO_DIR="${DATASETS_CUSTOM}/${myarray[-1]}"
 python bsmetadata/train.py \
   data_config.experiment="with_metadata" \
   data_config.metadata_config.metadata_list=["entity"] \
-  data_config.metadata_config.entity_setting = "beg" \
-  data_config.metadata_config.local_metadata_special_token_start = {"entity": "<ENTITY_CHAIN>"} \
-  data_config.metadata_config.local_metadata_special_token_end = {"entity": " <SUMMARY> "} \
+  data_config.metadata_config.entity_setting="beg" \
+  data_config.metadata_config.local_metadata_special_token_start={"entity": "<ENTITY_CHAIN>"} \
+  data_config.metadata_config.local_metadata_special_token_end={"entity": " <SUMMARY> "} \
```

Comment:
@manandey, just to highlight a change that fixed a part of the problem: we need to not put extra spaces in bash script for this kind of command